src/test/java/org/snomed/aag/rest/ServiceIntegrationControllerTest.java

```diff
@@ -252,37 +252,6 @@ void receiveCommitInformation_ShouldReturnExpectedResponse_WhenRequestingTwice()
 		assertResponseStatus(secondRequest, 409); // Project has been incremented and PAC hasn't been completed for that iteration.
 	}
 
-	@Test
-	void receiveCommitInformation_ShouldReturnExpectedResponse_WhenMetadataConfiguredForSubset() throws Exception {
-		// given
-		String requestUrl = receiveCommitInformation();
```

Comment:
The items associated with the ProjectAcceptanceCriteria come from one method which is already being tested. Testing the same logic elsewhere felt redundant so I removed the tests.